packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -6,13 +6,17 @@ import type { BaseSelectOption } from "../types";
 import { areObjectsFlatEqual } from "../utils/objects";
 import { getFirstInvalidType, transformValidityStateToObject } from "../utils/validity";
 
-export type CustomErrorType = string | FormErrorMessages;
+export type CustomMessageType = string | FormMessages;
 
 export type CustomValidityProp = {
   /**
    * Custom error message to show. Will only show up after the user has interacted with the input.
    */
-  customError?: CustomErrorType;
+  customError?: CustomMessageType;
+  /**
+   * Custom success message to show. Will only show up after the user has interacted with the input.
+   */
+  customMessage?: CustomMessageType;
```

Comment:
```suggestion
  /**
   * Success message to show. Will only show up after the user has interacted with the input.
   */
  successMessage?: CustomMessageType;
```

Is this property only intended to be the green success state? i yes, what do you think about renaming it like this so its clearer?